gph/python/ripser_interface.py

```diff
@@ -520,7 +540,9 @@ def ripser_parallel(X, maxdim=1, thresh=np.inf, coeff=2, metric="euclidean",
                                    return_generators)
 
     # Unwrap persistence diagrams
-    dgms = res.births_and_deaths_by_dim
+    # He must do a call now, because call a method of object ripserResults
+    # Instead of calling an attribute of the object
```

Comment:
I don't know, because for generators, we don't need to call a function.

Reply:
I don't fully understand.  My point is that once this is merged it does not matter what the code was like before, and it might just end up being confusing to keep this comment in the very long term (esp. for other developers).  Unless there is something confusing about the current code for *new* developers?

Reply:
What my comment is trying to address is that now when you want the `barcodes`. You call method `births_and_deaths_by_dim`. But when you need the generator, here, you call an attribute directy `flag_persistence_generators_by_dim`.

But maybe the issue is the text in the comment itself, maybe something like:

```suggestion
    # Barcodes must match the inner type of C++ core filtration value.
    # We call a method from the bindings that returns the barcodes as
    # numpy arrays with np.float32 type
```

If you think that it is not necessary, we can also remove the comment. I let it more for future developers if they ask themself why a method is call instead of calling the attribute directly.

Reply:
Aaah, I see!  The new comment is great, would you be happy to commit that?

Reply:
Sure !